ocaml/tests/test_vm_placement.ml

```diff
@@ -596,6 +596,15 @@ module Categorisation = struct
     ]
 end
 
+
+let () =
+  Suite_init.harness_init () ;
+  (* Alcotest hides the standard output of successful tests,
+     so we will probably not exceed the 4MB limit in Travis *)
+  Debug.log_to_stdout () ;
+  Alcotest.run "Test VM Placement suite" ["test", test]
+
+
```

Comment:
Why isn't this at the end of the file? there's a `test` being defined at line 642